src/components/card.svelte

```diff
@@ -0,0 +1,25 @@
+<div class="card">
+	<div class="card-inner">
+		<div class="card-header">
+			<slot name="card-header" />
+		</div>
+		<div class="card-body">
+			<slot name="card-body" />
+		</div>
+		<div class="card-footer">
+			<slot name="card-footer" />
+		</div>
+	</div>
+</div>
+
+<style lang="postcss">
+	.card {
+		@apply py-4 px-8 bg-white shadow-lg rounded-lg flex;
```

Comment:
there are just `@apply` decorators. This styling might move directly to the corresponding elements, to get rid of this css part.